src/Stackctl/StackSpecYaml.hs

```diff
@@ -129,8 +145,22 @@ parameterPairs (ParameterYaml k v) = [k .= v]
 newtype TagsYaml = TagsYaml
   { unTagsYaml :: [TagYaml]
   }
+  deriving stock (Eq, Show)
   deriving newtype ToJSON
 
+instance Semigroup TagsYaml where
+  TagsYaml as <> TagsYaml bs =
+    TagsYaml
+      $ map (TagYaml . uncurry newTag)
+      $ HashMap.toList
+      $ HashMap.fromList
```

Comment:
The tests confirm this does what we want (last-wins), but it's ticky. It relies on how `fromList` behaves and the order of `as <> bs` we're sending in. Something like `fromListWith (<>)` with a `Last` wrapper would be clearer. But that still relies on the `as <> bs` choice so :shrug: